packages/blockchain-wallet-v4-frontend/src/modals/Swap/PreviewSwap/index.tsx

```diff
@@ -168,7 +174,7 @@ class PreviewSwap extends PureComponent<InjectedFormProps<{}, Props> & Props> {
                     {coinToString({
                       value: convertBaseToStandard(
                         BASE.baseCoin,
-                        value ? value.fee : 0
+                        value ? this.networkFee(value) : 0
```

Comment:
One small optimization could be to move the `value` check inside of the `networkFee` method so you keep the else logic contained. So it would look like:

```
    value: convertBaseToStandard(
         BASE.baseCoin,
         this.networkFee(value),
    )
```

Reply:
not mission critical or anything

Reply:
Good callout, moved this out to the method.